addons/mail/static/src/new/public/discuss_public.js

```diff
@@ -30,6 +31,9 @@ export class DiscussPublic extends Component {
                         null,
                         `/discuss/channel/${this.thread.id}${window.location.search}`
                     );
+                    if (this.thread.defaultDisplayMode === "video_full_screen") {
+                        this.rtc.toggleCall(this.thread, true);
```

Comment:
we should have a way to enterFullScreen mode like before. But the problem is from rtc_services, I don't find a good way to put or call the `Call/enterFullScreen()`, previously, we have the model/component so we can call the model directly. But now, we don't have an access to the Call directly, so maybe we should have a prop like `enterWithFullScreen`? I don't know if @ThanhDodeurOdoo likes this or not.

Reply:
@zel-odoo wouldn't `this.props.thread.defaultDisplayMode` from the call component work?

Reply:
I don't think so, this is what we had before and I don't find a good alternative to make the call go into fullScreen
```js
await this.channel.toggleCall({ startWithVideo: true });
await this.threadView.callView.activateFullScreen();
```

Reply:
@zel-odoo maybe an event on the bus is the most straightforward